# Importing from __future__ after any other statement will cause an
# exception in Python 2.5 and newer.

diff -ru ../SOAPpy-0.11.6.orig/SOAPpy/Client.py ./SOAPpy/Client.py
--- ../SOAPpy-0.11.6.orig/SOAPpy/Client.py	2004-04-27 08:45:58.000000000 -0700
+++ ./SOAPpy/Client.py	2012-10-02 19:09:30.000000000 -0700
@@ -43,8 +43,6 @@
 ident = '$Id: Client.py,v 1.20 2004/04/10 04:22:52 irjudson Exp $'
 from version import __version__
 
-from __future__ import nested_scopes
-
 #import xml.sax
 import urllib
 from types import *
diff -ru ../SOAPpy-0.11.6.orig/SOAPpy/GSIServer.py ./SOAPpy/GSIServer.py
--- ../SOAPpy-0.11.6.orig/SOAPpy/GSIServer.py	2004-02-17 20:08:24.000000000 -0800
+++ ./SOAPpy/GSIServer.py	2012-10-02 19:09:37.000000000 -0700
@@ -46,8 +46,6 @@
 ident = '$Id: GSIServer.py,v 1.4 2004/02/04 03:21:02 irjudson Exp $'
 from version import __version__
 
-from __future__ import nested_scopes
-
 #import xml.sax
 import re
 import socket
diff -ru ../SOAPpy-0.11.6.orig/SOAPpy/NS.py ./SOAPpy/NS.py
--- ../SOAPpy-0.11.6.orig/SOAPpy/NS.py	2004-01-30 20:20:05.000000000 -0800
+++ ./SOAPpy/NS.py	2012-10-02 19:09:27.000000000 -0700
@@ -40,8 +40,6 @@
 ################################################################################
 """
 
-from __future__ import nested_scopes
-
 ident = '$Id: NS.py,v 1.3 2004/01/31 04:20:05 warnes Exp $'
 from version import __version__
 
diff -ru ../SOAPpy-0.11.6.orig/SOAPpy/Server.py ./SOAPpy/Server.py
--- ../SOAPpy-0.11.6.orig/SOAPpy/Server.py	2004-04-28 14:47:10.000000000 -0700
+++ ./SOAPpy/Server.py	2012-10-02 19:09:35.000000000 -0700
@@ -43,8 +43,6 @@
 ident = '$Id: Server.py,v 1.20 2004/04/28 21:47:10 warnes Exp $'
 from version import __version__
 
-from __future__ import nested_scopes
-
 #import xml.sax
 import re
 import socket
diff -ru ../SOAPpy-0.11.6.orig/SOAPpy/Types.py ./SOAPpy/Types.py
--- ../SOAPpy-0.11.6.orig/SOAPpy/Types.py	2004-09-10 20:03:33.000000000 -0700
+++ ./SOAPpy/Types.py	2012-10-02 19:09:25.000000000 -0700
@@ -36,8 +36,6 @@
 ident = '$Id: Types.py,v 1.17 2004/09/11 03:03:33 warnes Exp $'
 from version import __version__
 
-from __future__ import nested_scopes
-
 import UserList
 import base64
 import cgi
